packages/api/routes/index.js

```diff
@@ -10,4 +11,12 @@ router.get('/nomination/:id', nominationController.getNominationById);
 // user endpoints
 router.get('/user/:id', userController.getUserById);
 
+// grant cycle endpoints
+router.get('/grantcycle/findall', grantCycleController.findAll);
+router.get('/grantcycle/findactive', grantCycleController.findActive);
+router.post('/grantcycle/findbyname', grantCycleController.findByName);
```

Comment:
first makes sense

second: I thought that was part of the requirement from Tina, but I'll ask again to make sure

Reply:
As a note, Pavel and I talked through `/findbyname` and we discussed changing it to be a `GET` route opposed to a `POST` route as well as changing to url route parameters. The one concern that was brought up that the name could contain a space. If it is a route parameter, then we are to assume that it will be HTML encoded to be valid so we will want to [decode the route param](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURI)